pkg/scanner/utils/utils_test.go

```diff
@@ -0,0 +1,80 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/knqyf263/go-version"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMatchVersions2(t *testing.T) {
+	testCases := []struct {
+		name           string
+		currentVersion string
+		rangeVersion   []string
+		expectedCheck  bool
+	}{
+		{
+			name:           "pass: expect true when os/machine is in pre-release",
+			currentVersion: "1.9.25-x86-mingw32",
+			rangeVersion:   []string{`>= 1.9.24`},
+			expectedCheck:  true,
+		},
+		{
+			name:           "pass: expect true when language is in pre-release",
```

Comment:
Do you think `-java` is pre-release? It looks just a kind to me.
https://rubygems.org/gems/ffi/versions

I meant we might need to change the comment. I think we don't need to change the logic.

Reply:
Done